src/main/java/mate/model/Driver.java

```diff
@@ -5,13 +5,21 @@
 public class Driver {
     private Long id;
     private String name;
+    private String login;
+    private String password;
     private String licenseNumber;
 
     public Driver(String name, String licenseNumber) {
```

Comment:
I think we should add `toString()` method here(to display valid data in exception messages)